modular_citadel/code/modules/projectiles/guns/energy/energy_gun.dm

```diff
@@ -1,13 +1,6 @@
 /obj/item/gun/energy/e_gun
 	name = "blaster carbine"
 	desc = "A high powered particle blaster carbine with varitable setting for stunning or lethal applications."
-	icon = 'modular_citadel/icons/obj/guns/OVERRIDE_energy.dmi'
```

Comment:
This reverts all egun sprites back to their original sprites, which makes certain eguns invisible. Either copy the changed and added sprites from the override dmi to the original dmi or add the changed egun sprite to the override dmi